src/main/java/com/ait/lienzo/client/widget/panel/event/LienzoPanelScrollEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.ait.lienzo.client.widget.panel.event;
+
+import com.ait.lienzo.client.widget.panel.AbstractLienzoPanelEvent;
+
+public class LienzoPanelScrollEvent
+        extends AbstractLienzoPanelEvent<LienzoPanelScrollEventHandler>
+{
+    public static final Type<LienzoPanelScrollEventHandler> TYPE = new Type<LienzoPanelScrollEventHandler>();
```

Comment:
Explicit type argument `LienzoPanelScrollEventHandler` can be replaced with <>

Reply:
- LienzoPanelImpl